src/ucd-data-fetch.c

```diff
@@ -111,6 +111,16 @@ static struct cloud_struct config[MAX_CONFIGS] = {
 		"  - name: aliyun\n" \
 		"    groups: wheelnopw\n" \
 		"ssh_authorized_keys:\n"
+	},
+	{
+		"equinix",
+		"192.80.8.124",
```

Comment:
Maybe  rebase this so it comes after the hostname lookup patch and this IP address isn't needed in here anymore.